README.md

```diff
@@ -18,21 +18,52 @@
 
 ## :rocket: Getting started
 
-### Install Iter8 CLI
+### Using Iter8 Action
+
+Configure experiment with Helm values file and call:
+
+``` yaml
+    - run: |
+        cat << EOF > expConfig.yaml
+          url: https://httpbin.org/get
+          SLOs:
+            http/error-rate: 0
+            http/mean-latency: 100
+        EOF
+    - uses: iter8-tools/iter8@v0.9
```

Comment:
There doesn't seem to be any connection between `iter8-tools/iter8@v0.9` used here and `iter8/iter8:0.9` used in the Docker file.

***

This is what we want to connect up, so that when we go to v0.10.0 of Iter8 CLI, the GH action can be upgraded easily/automatically.

Reply:
By using action v0.9 one is using that github ref to fetch the dockerfile. So they are connected. Changing the ref in the dockerfile means one will need to change the action ref too. i would prefer less of a relationship so less places to change when changing version but I don't see it.

Reply:
> By using action v0.9 one is using that github ref to fetch the dockerfile.

How? The Dockerfile ref is hardcoded and doesn't seem dependent on the `v0.9` tag -- which at the moment is not even a valid GitHub ref for the Iter8 repo.

Reply:
Perhaps the simplest solution is to say:

```
- uses: iter8-tools/iter8 # you can also replace this with iter8-tools/iter8@tag where tag is any valid Iter8 release tag
```

Reply:
List of valid Iter8 tags:  https://github.com/iter8-tools/iter8/tags
List of valid Iter8 branches: https://github.com/iter8-tools/iter8/branches

Reply:
> But you need a valid GitHub ref to accomplish the above. v0.9 is not that ref ATM, because Iter8 doesn't have this ref.

Ah. My mistake. This is an issue. I suppose we can create a branch and keep it up to date on release. But this is a significant complexity.